README.md

```diff
@@ -17,10 +17,12 @@
 
 ## Quick start
 
-With stylelint, it's easy to start linting your CSS to enforce your preferred conventions:
+With stylelint, it's easy to start linting your CSS:
 
 1. Install stylelint: `npm install stylelint`.
-2. Learn about [some rules](/docs/user-guide/rules.md). No rules are turned on by default, so you only have to learn about the rules you want to enforce; and you can start small, growing your config over time as you have a chance to explore more of the rules.
+2. Choose whether you want to craft your own config from the ground up, or use a pre-written one:
+  * To craft your own: learn about [some rules](/docs/user-guide/rules.md). No rules are turned on by default, so you only have to learn about the rules you want to enforce; and you can start small, growing your config over time as you have a chance to explore more of the rules.
+  * To use a pre-written one: we recommend using  [`stylelint-config-suitcss`](https://github.com/stylelint/stylelint-config-suitcss)... it'll get you going with plenty of sensible defaults.
 3. Create your [configuration](/docs/user-guide/configuration.md), probably as a `.stylelintrc` file.
```

Comment:
This is a good change, but I think it is not being explicit enough about another option: copy-paste from `stylelint-config-suitcss` in order to write your own config without having to start from scratch. What do you think about this wording?

Choose whether you want to craft your own config or extend a pre-written, shared config. 
  * If you want to use a pre-written config, just find one and extend it. We recommend trying [`stylelint-config-suitcss`](https://github.com/stylelint/stylelint-config-suitcss), which includes all of stylelint's rules with sensible defaults. (You can always override specific rules after extending a config.)
  * To craft your own from the ground up, just learn about [some rules](/docs/user-guide/rules.md). *All of the rules are off by default*, so you only have to learn about the rules you want to turn on and enforce. That way you can start small, growing your config over time as you have a chance to explore more of the rules. Alternately, you could copy-paste the configuration in `stylelint-config-suitcss` in order to start with defaults for all the rules, then customize from there.